bagelbids/tests/test_cli_pheno.py

```diff
@@ -85,6 +92,50 @@ def test_get_columns_that_are_about_concept(test_data):
     assert [] == get_columns_about(data_dict, concept="does not exist concept")
 
 
+def test_map_columns(test_data):
+    """Test that inverse mapping of concepts to columns is correctly created"""
+    with open(test_data / "example2.json", "r") as f:
+        data_dict = json.load(f)
+
+    result = map_categories_to_columns(data_dict)
+
+    assert {"participant", "session", "sex"}.issubset(result.keys())
+    assert ["participant_id"] == result["participant"]
+    assert ["session_id"] == result["session"]
+    assert ["sex"] == result["sex"]
+
+
+def test_get_transformed_categorical_value(test_data):
+    """Test that the correct transformed value is returned for a categorical variable"""
+    with open(test_data / "example2.json", "r") as f:
+        data_dict = json.load(f)
+
+    assert "bids:Male" == get_transformed_values(
+        columns=["sex"],
+        row=pd.Series({"sex": "M"}, index=["sex"]),
+        data_dict=data_dict,
+    )
+
+
+def test_missing_values():
+    """Test that missing values are correctly detected"""
+    test_data_dict = {
+        "test_column": {"Annotations": {"MissingValues": ["test_value"]}},
+        "empty_column": {"Annotations": {}},
+    }
+
+    assert (
```

Comment:
Can parameterize these statements as well in a refactor.

Reply:
Yeah, absolutely. In fact I think I'll just do it here quickly